src/test/java/emailvalidator4j/parser/DomainPartTest.java

```diff
@@ -54,7 +55,7 @@ public void invalidPartThrowsException(Class type, String domainPart) throws Inv
                 {DomainNotAllowedCharacter.class, "@example\\"},
                 {DomainNotAllowedCharacter.class, "@exa\\mple"},
                 {UnclosedDomainLiteral.class, "@example]"},
-                {ConsecutiveGeneric.class, "@example'"},
+                {DomainNotAllowedCharacter.class, "@example'"},
```

Comment:
The failure mode here is more specific now. Consecutive generic was the best I could think of before and it wasn't very good.